app/views/admin/versions/_event_change_description.html.haml

```diff
@@ -0,0 +1,35 @@
+- if version.event == 'create'
```

Comment:
Sorry :) .
Trying to use case like you suggested threw up this [RuboCop offence](http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/EmptyCaseCondition)